### PR TITLE
Update capacity-scheduling.md

### PR DESCRIPTION
**Pull request by googs1025**
- fix some resource typo

### CODE REVIEW

File: docs/user-manuals/capacity-scheduling.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ If you want to use multi quota trees, you need to set feature-gate `MultiQuotaTr
 
 ### Quick Start by Label
 
-1.Create a Deployment `quota-example` with the YAML file below.
+1.Create a ElasticQuota `quota-example` with the YAML file below.
```
Comment (saintube) [inline docs/user-manuals/capacity-scheduling.md:37]: Nice catch! Besides, `"a ElasticQuota"` -> `"an ElasticQuota"`

Comment (googs1025) [reply docs/user-manuals/capacity-scheduling.md]: done

Comment (googs1025) [reply docs/user-manuals/capacity-scheduling.md]: done

**Comment by googs1025** [comment]
friendly ping @eahydra @hormes  :)

**Review by saintube — APPROVED** [review]

**Pull request merged**